song_translator/track/utils.py

```diff
@@ -105,3 +105,11 @@
     ('yo', 'yoruba'),
     ('zu', 'zulu'),
 )
+lang_short = ['af', 'sq', 'am', 'ar', 'hy', 'az', 'eu', 'be', 'bn', 'bs', 'bg', 'ca', 'ceb', 'ny', 'co',
```

Comment:
still duplication. could you extract list of tuples from Choices constructor to a variable and then use it for both `lang_re` and constructor of Choices?